### PR TITLE
Update deeplink-url-request.markdown

### PR DESCRIPTION
**Pull request by f80**
Fix some glitch in display

### CODE REVIEW

File: src/api-guides/request/deeplink-url-request.markdown [hunk 1]
```diff
@@ -28,9 +27,9 @@ https://www.concursolutions.com/goto/travel-request/{requestUuid}?companyUUID={c
 
 |Name|Type|Format|Description|
 |---|---|---|---|
-`requestUuid`|`string`|`path`|**Required** The 20-character unique identifier of the request that is being read.|
-`companyUUID`|`string`|`query`|The unique identifier of the company, in UUID format, to which the request belongs to.|
-`context`|`string`|`query`|The access level of the user, which determines the form fields they can view/modify. Supported values: `TRAVELER`, `MANAGER`, or `PROCESSOR`|
+|`requestUuid`|`string`|`path`|**Required** The 20-character unique identifier of the request that is being read.|
```
Comment (laurentlhermine) [inline src/api-guides/request/deeplink-url-request.markdown:30]: Don't we have 32 characters ?

Comment (f80) [reply src/api-guides/request/deeplink-url-request.markdown]: You're right, I changed it.

**Review by marcdalleau — APPROVED** [review]

**Review by laurentlhermine — APPROVED** [review]

File: src/api-guides/request/deeplink-url-request.markdown [hunk 2]
```diff
@@ -28,9 +27,9 @@ https://www.concursolutions.com/goto/travel-request/{requestUuid}?companyUUID={c
 
 |Name|Type|Format|Description|
 |---|---|---|---|
-`requestUuid`|`string`|`path`|**Required** The 20-character unique identifier of the request that is being read.|
-`companyUUID`|`string`|`query`|The unique identifier of the company, in UUID format, to which the request belongs to.|
-`context`|`string`|`query`|The access level of the user, which determines the form fields they can view/modify. Supported values: `TRAVELER`, `MANAGER`, or `PROCESSOR`|
+|`requestUuid`|`string`|`path`|**Required** The 32-character unique identifier of the request that is being read.|
+|`companyUUID`|`string`|`query`|The unique identifier of the company, in UUID format, to which the request belongs to.|
```
Comment (laurentlhermine) [inline src/api-guides/request/deeplink-url-request.markdown:31]: it is possible to avoid mixing spelling with `Uuid` and `UUID`

**Review by RoAnnC1 — APPROVED** [review]

**Comment by RoAnnC1** [comment]
:shipit:

File: src/api-guides/request/deeplink-url-request.markdown (hunk 2 above)
Comment (RoAnnC1) [reply src/api-guides/request/deeplink-url-request.markdown:31]: @laurentlhermine I can fix this for you.

**Pull request merged**